src/plugins/kibana_usage_collection/server/collectors/event_loop_delays/saved_objects.ts

```diff
@@ -35,10 +35,18 @@ export function registerSavedObjectTypes(registerType: SavedObjectsServiceSetup[
   });
 }
 
-export function serializeSavedObjectId({ date, pid }: { date: moment.MomentInput; pid: number }) {
+export function serializeSavedObjectId({
+  date,
+  pid,
+  instanceUuid,
+}: {
```

Comment:
Hum, is this really `instanceUuid` we want to use there? From my understanding (@lukeelmers please correct me if I'm wrong) all workers of the instance will have the same instance uuid, and a new worker-uuid (or something) will be introduced to dissociate the processes?

Reply:
workers will have different pids so each SO will have a `unique` id across workers. The problem `instanceUuid` is trying to solve is that docker will assign `pid 1` for the main kibana process across different docker containers.

So if we have a multi-kibana cluster then they'll all have the same `pid`. the `instanceUuid` will insure that even if the kibana instances are assigned the same pid (by change or inside a docker container) the ids will remain unique.